dev-docs/security-overview.md

```diff
@@ -132,7 +132,7 @@ The CLI uses this connection for two essential operations at the Kubernetes leve
 1. It executes the [hardcoded Helm charts](#cli-root-of-trust), which, most notably, install the three core services KeyService, JoinService, and VerificationService, the [constellation-node-operator](https://github.com/edgelesssys/constellation/tree/main/operators/constellation-node-operator), and a small number of standard services like Cilium and cert-manager.
 
 The latter causes the first node to download, verify, and run the containers defined in the Helm charts.
-The containers that are specific to  Constellation are hosted at https://ghcr.io/edgelesssys.
+The containers that are specific to  Constellation are hosted at ghcr.io/edgelesssys.

```

Comment:
Maybe even quote with backticks.

Reply:
Yeah, I tried it but wasn't sure what the link checker does with that, since VSCode identified and "linkyfied" it in the editor.

Reply:
Ah I think you mean additionally, did that now.